src/setuptools_scm/config.py

```diff
@@ -49,6 +49,7 @@ class Configuration(object):
     parse = None
     _tag_regex = None
     _absolute_root = None
+    enabled = False
```

Comment:
this needs to be handled in the legacy integration (setup_keyword)

since 
```
In [2]: toml.loads("[foo.bar]")                                                                                                                                                                                                                              
Out[2]: {'foo': {'bar': {}}}
```
i believe we should use the existence of the section as indicator for trigger rather than introducing a extra boolean

Reply:
> i believe we should use the existence of the section as indicator for trigger rather than introducing a extra boolean

Yes, I think I agree with that. I do wish to avoid booleans in general.

My reasoning behind having the boolean was that I was imagining some users might have `setuptools_scm` present and some configuration present for a number of reasons:

- they have a draft of the config but aren't yet ready to enable it
- they want to control some other aspect of setuptools_scm not related to versioning (file finding or maybe some future functionality)

Basically, I worry that if the section is `tool.setuptools_scm`, that's to general to imply "infer a version from scm metadata". If the section were `tool.setuptools_scm.infer_version`, I would agree whole-heartedly.

Reply:
as the key feature of the package is version finding from scm metadata,
if people want to write up a draft and "hide it", they can use `tool._setuptools_scm`

in this case i believe that trying to account for non-existing or currently non-configurable features would constitute a premature optimization

should a new need arise, it needs to get a new name in the tool section

Reply:
Done in bf15ea9.